WebRoot/js/zimbraAdmin/globalconfig/controller/ZaGlobalConfigViewController.js

```diff
@@ -377,6 +377,42 @@ function () {
 			mods[ZaGlobalConfig.A_zimbraMtaBlockedExtension] = "";
 		}		
 	}
+
+	//Check validation of reverse proxy upstream imap server: host name or IP address format
+	var invalidValues = [];
+	var isValidValue = true;
+	if(ZaItem.hasWritePermission(ZaGlobalConfig.A_zimbraReverseProxyUpstreamImapServers,tmpObj)) {
```

Comment:
is this code same as ZaServerController.prototype.validateReverseProxyUpstreamImapServers ?

Reply:
No, they are different .

Reply:
Can you please explain in detail what is different here?

Reply:
ZaItem.hasWritePermission checks permission of account modified this attribute and check this attribute can be modified or not. 
ZaServerController.prototype.validateReverseProxyUpstreamImapServers check validation of parts of input value.